created_id.gemspec

```diff
@@ -27,7 +27,7 @@ Gem::Specification.new do |spec|
   spec.require_paths = ["lib"]
 
   spec.add_dependency "activerecord", ">= 5.0"
-
+  spec.add_dependency "railties", ">= 5.0"
```

Comment:
Since we don't need the generator, this line can be removed so that this continue being a soft dependency. This allows the gem to work outside of a Rails environment.

Reply:
Removed